CodeStarterCamp_Week3/main.swift

```diff
@@ -7,5 +7,5 @@
 
 import Foundation
 
-yagombucks.takeOnOrder(Coffee.moca)
+missKim.purchaseCoffee(Coffee.moca, of: yagombucks, by: missKim.name)
```

Comment:
요기도 매개변수 타입을 지정했기 때문에 타입은 생략이 가능합니다! 😁
```suggestion
missKim.purchaseCoffee(.moca, of: yagombucks, by: missKim.name)
```

Reply:
해당내용도 수정완료 하였습니다.